File: SoloLearn Codes/How Far/HowFar.cs
using System;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            string s = Console.ReadLine();
            Console.WriteLine(Math.Abs(s.IndexOf("H") - s.IndexOf("P")) - 1);
        }
    }
}

File: SoloLearn Codes/How Far/HowFar.java
import java.util.Scanner;

public class HowFar
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String street = sc.nextLine();
        System.out.println(Math.abs(street.indexOf('H') - street.indexOf('P')) - 1);
    }
}